src/mount_efs/__init__.py

```diff
@@ -329,6 +335,15 @@ def get_aws_security_credentials(use_iam, awsprofile=None, aws_creds_uri=None):
         if credentials and credentials_source:
             return credentials, credentials_source
 
+    if WEB_IDENTITY_ROLE_ARN_ENV in os.environ and WEB_IDENTITY_TOKEN_FILE_ENV in os.environ:
```

Comment:
Can you add a comment above on how this piece of code attempts to retrieve credentials?

Reply:
Done.